galley/pkg/crd/validation/webhook.go

```diff
@@ -41,6 +41,7 @@ import (
 	"istio.io/istio/pilot/pkg/config/kube/crd"
 	"istio.io/istio/pilot/pkg/model"
 	"istio.io/istio/pkg/log"
+	"k8s.io/kubernetes/staging/src/k8s.io/client-go/scale/scheme/extensionsv1beta1"
```

Comment:
This doesn't look right, but I can't find any code that is actually referencing it in this file? Stale import?